csi/crypto/crypto.go

```diff
@@ -96,8 +117,14 @@ func OpenVolume(volume, devicePath, passphrase string) error {
 
 // CloseVolume closes encrypted volume so it can be detached.
 func CloseVolume(volume string) error {
+	namespaces := []lhtypes.Namespace{lhtypes.NamespaceMnt, lhtypes.NamespaceIpc}
+	nsexec, err := lhns.NewNamespaceExecutor(lhtypes.ProcessNone, lhtypes.ProcDirectory, namespaces)
+	if err != nil {
+		return err
+	}
+
 	logrus.Infof("Closing LUKS device %s", volume)
-	_, err := luksClose(volume)
+	_, err = nsexec.LuksClose(volume, lhtypes.LuksTimeout)
 	return err
```

Comment:
If the code like below, it will be perfect.

```go
	logrus.Infof("Closing LUKS device %s", volume)
	_, err := luksClose(volume)
	_, err = nsexec.LuksClose(volume, lhtypes.LuksTimeout)
	return err
```

Reply:
Why do we want to call the operation twice?

https://github.com/longhorn/longhorn-manager/blob/v1.5.3/csi/crypto/luks.go#L24
```go
func luksClose(volume string) (stdout string, err error) {
	return cryptSetup("luksClose", volume)
}
```

https://github.com/longhorn/go-common-libs/blob/main/ns/crypto.go#L18
```go
func (nsexec *Executor) LuksClose(volume string, timeout time.Duration) (stdout string, err error) {
	args := []string{"luksClose", volume}
	return nsexec.Cryptsetup(args, timeout)
}
```

Reply:
> Why do we want to call the operation twice?
Typos ..